sql/2020/09_Performance/performance_observer.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
+# Percent of pages using Performance observer
+  
+SELECT
+  client,
+  COUNT(0) AS total_pages,
+  COUNTIF(performance_pages > 0) AS performance_pages,
+  COUNTIF(performance_pages > 0) / COUNT(0) AS pct_performance_pages,
+FROM (
+  SELECT
+    client,
+    COUNTIF(body LIKE '%new PerformanceObserver%') AS performance_pages,
+  FROM
+    `httparchive.almanac.summary_response_bodies`
+  WHERE
+    date = "2019-08-01"
+    AND type = "script"
+  GROUP BY
+    client,
+    page)
+GROUP BY
+  client
```

Comment:
Querying response_bodies is really expensive. Fortunately, there's a use counter for this specific API: https://www.chromestatus.com/metrics/feature/timeline/popularity/2188. We can query it like this: 
```suggestion
SELECT
  client,
  num_urls,
  total_urls,
  pct_urls
FROM
  `httparchive.blink_features.usage`
WHERE
  yyyymmdd = '20200801' AND
  feature = 'PerformanceObserverForWindow'
```